resources/authgear/templates/en/web/settingsv2/settings.html

```diff
@@ -0,0 +1,155 @@
+{{ template "settingsv2/__wide_page_frame.html" . }}
+
+{{ define "page-content" }}
+
+<div class="flex flex-col gap-8">
+  <div class="flex flex-col gap-4 items-center">
+    <img
+    class="rounded-full bg-center bg-cover bg-no-repeat w-30 h-30"
+    {{ if (and (call $.IsReadable "/picture") $.Picture) }}
+      src="{{ $.Picture }}"
+    {{ else }}
+      src="{{ call $.GeneratedStaticAssetURL "v2_avatar_placeholder.svg" }}"
+    {{ end }}
+    >
+    <div>
+      <div class="settings-label">
+        {{ $any_name_readable := (or (call $.IsReadable "/name") (call $.IsReadable "/given_name") (call $.IsReadable "/family_name") (call $.IsReadable "/middle_name") (call $.IsReadable "/nickname")) }}
+        {{ if (and $any_name_readable $.FormattedName) }}<span class="truncate">{{ $.FormattedName }}</span>{{ end }}
+        {{ if (and (not $.FormattedName) $.EndUserAccountID) }}<span class="truncate">{{ $.EndUserAccountID }}</span>{{ end }}
+      </div>
+      <p class="body-text--lg">
+        {{ $editProfileHref := call $.MakeURL "/settings/v2/profile"}}
+        {{ template "v2-settings-edit-profile" (dict "href" $editProfileHref )}}
+      </p>
+
+    </div>
+  </div>
+
+  {{/* Identity */}}
+  <div class="flex flex-col gap-2">
+    <div class="flex flex-col gap-1">
+      <div class="settings-title text-start">
+        {{ template "v2-settings-my-account" . }}
+      </div>
+      <div class="settings-description text-start">
+        {{ template "v2-settings-my-account-description" . }}
+      </div>
+    </div>
+
+    <div>
+    {{ range .IdentityCandidates }}
+    {{ $is_unset := (not .identity_id) }}
+    {{ if eq .type "login_id" }}
+    {{ if eq .login_id_type "email" }}
+      <a
+        class="settings-btn w-full row-sep"
+        {{ if .identity_id }}
+          href="{{ call $.MakeURL "/settings/v2/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type "q_identity_id" .identity_id }}"
+        {{ else }}
+          href="{{ call $.MakeURL "/settings/v2/enter_login_id" "q_login_id_key" .login_id_key "q_login_id_type" .login_id_type "q_login_id_input_type" .login_id_input_type }}"
+        {{ end }}
+      >
+        <div class="grid grid-cols-3 gap-y-2 w-full">
```

Comment:
Wrong layout.
The design is not divde the row into 3 columns with same width.

It is

```
|---176px---|---Fill remaining spce---|---18px---|
```